docs/userguide/security-governance/working-with-gatekeeper.md

```diff
@@ -407,7 +407,7 @@ In this case, you will use Gatekeeper v3.7.2. Related deployment files are from
      name: kubeconfig
      namespace: gatekeeper-system
    ``` 
-   You need to fill in the dummy pod created in step 1 to {{ POD_NAME }} and fill in the secret which represents kubeconfig pointing to karmada-apiserver.
+   You need to fill in the dummy pod created in step 1 to \{\{ POD_NAME \}\} and fill in the secret which represents kubeconfig pointing to karmada-apiserver.
```

Comment:
I think yes, if we want to update to version 3.
As for me, you just need to wrap these variables {{ POD_NAME }} in something \*\*\*{{ POD_NAME }}\*\*\* for example or other ``` characters.

And it would be good to do something about this https://github.com/karmada-io/website/pull/565#issuecomment-1950703873
Because it could be a problem.

Reply:
Perhaps we should integrate a syntax detection command into git commit, allowing users to identify issues promptly and receive appropriate solutions, thereby preventing collective confusion.

MDX3 appears to be more stringent than Markdown, and it's unclear whether this is advantageous.

Reply:
\`\` seems more common in markdown like:

- Example 1: `{{ POD_NAME }}`

- Example 2:

  ```
  CLUSTER_NAME=<member_cluster_name>
  HOST_IP=<member_cluster_entry_ip>
  PORT=<member_cluster_entry_port>
  KUBECONFIG=<member_cluster_kubeconfig>
  ```